src/main/java/com/rocketden/main/service/RoomService.java

```diff
@@ -27,6 +27,7 @@
 
     public static final int ROOM_ID_LENGTH = 6;
     public static final long MAX_DURATION = 3600; // 1 hour
+    public static final long MAX_SIZE = 30;
```

Comment:
nit: change to to be an int (I think the reason duration is a long is because that's what the timer API expects, but size can just be an int)